### PR TITLE
Move tests part2

### PR DESCRIPTION
**Pull request by ishiy1993**

### CODE REVIEW

File: runtime/tests/test_cast.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pytest
+from onnion_runtime import Cast
+
+from .utils import check, on_arm32
+
+
+def test_cast_00():
+    opset_version = 13
+    t = 1  # FLOAT
```
Comment (koiking213) [inline runtime/tests/test_cast.py:10]: What is the reason you don't use `onnx.TensorProto`, like legacy test?

Comment (ishiy1993) [reply runtime/tests/test_cast.py:10]: `onnx` is not installed in `idein/onnion-arm32-env`. 
[`python3-onnx`](https://packages.debian.org/bullseye/python3-onnx) is old (version 1.7) and the installation by pip is failed.

Comment (koiking213) [reply runtime/tests/test_cast.py:10]: Thanks, I got it.

**Review by koiking213 — APPROVED** [review]
LGTM

**Pull request merged**